### DISCUSSION TITLE
Instructions for retrieving specific translation for a read with the JS SDK

### DISCUSSION THREAD

**Discussion by dvodonnell** [directus/directus#4964] [Archive]
Can we get the docs updated to show an example of how one might do a readMany query with the JS SDK that retrieves the value of translated fields, ideally with a locale specified?

By default the result just returns an array with the number of translations available, and it's unclear how to get more than that. For users who are making use of translated fields out of the box this is a pretty significant non-starter.

Thank you!

**Comment by rijkvanzanten** [comment]
See https://docs.directus.io/reference/api/introduction/#relational-data and https://docs.directus.io/reference/api/query/#fields and https://docs.directus.io/reference/sdk/#read-by-query 🙂 

TL;DR, use:

```js
readMany({
  fields: ['*.*']
});
```

**Reply by dvodonnell** [reply]
Great, thanks!

Maybe still worth explicitly making reference to translations as part of the API docs anyway? Even though I suppose they are relational by nature, it's not obvious unless digging in to the table structure that this is the case, and they are treated a little differently than a standard relational field in terms of the UI and payload signature.

**Reply by benhaynes** [reply]
I don't think this should go in the SDK Docs, since that should reference the main docs. Is there a good place in the main docs that you think this should go?
